docs/tsm-report.cddl

```diff
@@ -1,8 +1,15 @@
 tsm-report = {
-  ? "auxblob": bstr
-  "outblob": bstr
-  "provider": bstr
+  ? auxblob: binary-string
+  outblob: binary-string
+  provider: tstr
   ? service-report
 }
-service-report //= ("manifestblob": bstr, "service_provider": bstr)
-service-report //= ("service_provider": bstr)
+
+service-report = ((
+  manifestblob: binary-string
+  service_provider: tstr
```

Comment:
Hi. service_provider is a WO attribute. So, how can we get the service_provider from TSM?

https://www.kernel.org/doc/Documentation/ABI/testing/configfs-tsm

Reply:
Yikes, well, that can be included as part of the attestation scheme I guess. The manifest being write only seems like a bug on the kernel side though..

Reply:
Perhaps we should hold off on registering this type before we can test it

Reply:
@deeglaze could you test it?